stan/math/prim/prob/frechet_lpdf.hpp

```diff
@@ -81,7 +81,7 @@ return_type_t<T_y, T_shape, T_scale> frechet_lpdf(const T_y& y,
                 T_partials_return, T_y>
       inv_y(size(y));
   for (size_t i = 0; i < stan::math::size(y); i++) {
-    inv_y[i] = 1.0 / value_of(y_vec[i]);
```

Comment:
The problem here was this second value_of (inv_y has already had one value_of).

Reply:
Yeah whichever goes first is fine with me.

Reply:
Now its in and you have a merge conflict.